### ISSUE TITLE
Derby and Mysql schemas

### ISSUE THREAD

**Issue by tngTUDOR** [GreenDelta/olca-modules#8]
a direct conversion of the derby schema:
+ olca-modules/olca-core/src/main/resources/org/openlca/core/database/internal/current_schema_derby.sql 
following the substitutions suggested on top of the file, do not produce a file identical to: 
+ olca-modules/olca-core/src/main/resources/org/openlca/core/database/internal/current_schema_mysql.sql

Using directly the olca-modules/olca-core/src/main/resources/org/openlca/core/database/internal/current_schema_mysql.sql to initialize a remote mysql database for example, will not work with release 1.5

```bash
sed -e "s/CLOB(64 K)/TEXT/" -e "s/SMALLINT default 0/TINYINT default 0/" -e "s/BLOB(16 M)/MEDIUMBLOB/" current_schema_derby.sql > current_schema_mysql_derived.sql
diff current_schema_mysql.sql current_schema_derby.sql > schemas_mysql.diff
```

[schemas_mysql.diff.txt](https://github.com/GreenDelta/olca-modules/files/640037/schemas_mysql.diff.txt)

**Comment by msrocka** [comment]
Thank you Tomas; I merged your pull request but will keep this issue open because it would be good to add such a script directly to the repo (probably under `olca-core/scripts`). However, we are not sure how we should continue with the MySQL support in the core APIs and desktop app as it seems that it is used only in special cases.

**Comment by tngTUDOR** [comment]
Yes, I thought a script would have been better too, maybe even a git hook. 
We really like using MySQL from our side, so we can only say: please keep it :+1:
